install-sddm-theme.sh

```
@@ -56,13 +59,32 @@ echo "⣿⣿⣿⣿⣿⣿⣿⣮⡛⣤⣀⣀⡠⣊⠤⠂⠀⠀⢹⠀⠀⠀⢇⠀
-        case $os in
-            Ubuntu|Kubuntu|Arch ) installPackages $os; break;;
-            * ) echo "Invalid selection. Please try again.";;
-        esac
-    done
+    # Auto detect OS
+    if grep -q "Kubuntu" /etc/os-release; then
+        echo "Auto detected distro: Kubuntu"
+        installPackages Kubuntu
+    elif grep -q "Fedora" /etc/os-release; then
+        echo "Auto detected distro: Fedora"
+        if [[ ! dnf repolist | grep -q "rpmfusion-free" ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)